interfaces/builtin/snap_interfaces_requests_control.go

```diff
@@ -0,0 +1,47 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2024 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const snapPromptingControlSummary = `allows use of snapd's prompting API and access to prompting-related notice types`
+
+const snapPromptingControlBaseDeclarationPlugs = `
```

Comment:
This is a nitpick for a follow up PR, according to https://github.com/canonical/snapd/blob/master/interfaces/builtin/README.md#base-declaration-policy-patterns the plug declaration can be removed entirely.

The same can be said for other similae interfaces `snap-refresh-observe` and `snap-themes-control`.

Reply:
I think this might change when we add the `"handler-service"` attribute in the follow-up PR, so I'll make sure to request you :)

Reply:
this isn't true because allow-installation is only for the relevant-side